static/js/runtime-config.js

```diff
@@ -4,7 +4,7 @@
 export default class RuntimeConfig {
   constructor (initialConfig = {}) {
     this._data = { ...initialConfig };
-    this._onUpdateListener = () => {};
+    this._listeners = [];
```

Comment:
would it make sense for this to be a map of keys to listeners? I think the SDK just has an array of objects but the listener lookup might be slightly faster if we use a map (though would probably be uglier to read and also pretty negligible). The keyless listeners would probably have to go into their own variable if we use a map instead (or be set to like the blank string key).